mantidproject/mantid
# Enable FitPeaks to "observe" more profiles and export effective parameters

Enable FitPeaks to observe peak width for more peak profile types at least including Lorentzian, PseudoVoigt and Voigt.  Besides, FitPeaks shall be able to export effective peak parameters alternative to raw peak parameters.

**To test:**

1. Check the new added unit test
2. Make sure all the unit tests and system tests are passed
3. Use FitPeaks to fit some Voigt/PseudoVoigt/Lorentzian peaks
4. Check the output with effective peak parameters

Fixes #22435.

*Does not need to be in the release notes, because the newly added features are standard features of FindPeaks, which FitPeaks is designed to replace.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
